app/views/miq_request/_prov_field.html.haml

```diff
@@ -48,6 +48,16 @@
     .form-group
       %label.col-md-2.control-label{:valign => "top"}
         = field_hash[:description]
+        - if field_hash[:help]
+          %a{:tabindex => 0,
+             :role             => "button",
+             :"data-toggle"    => "popover",
+             :"data-trigger"   => "focus",
+             :"data-html"      => "true",
+             :title            => "",
+             :"data-content"   => "#{field_hash[:help]}",
+             :"data-placement" => "top"}
+            %span.fa.fa-info-circle
```

Comment:
I'd say use your judgement :) if you feel like one is somehow dirty and a bad example that nobody should see, feel free to squash, just, don't default to squashing without thinking about it :)

Personally, in this case, I would indeed keep them separate, since the first one shows how you added a help button, while the other moves changes the intiatialization to a right place. As in, I consider the second one much clearer by itself, than if they were together.